demos/conanfile.py

```diff
@@ -23,14 +23,14 @@ class demos(ConanFile):
     default_options = {"platform": "unspecified"}
 
     def build_requirements(self):
-        self.tool_requires("cmake/3.27.1")
+        # self.tool_requires("cmake/3.27.1")
```

Comment:
```suggestion
        self.tool_requires("cmake/3.27.1")
```
We do need cmake to build :)

Reply:
Old debug code again.